### PR TITLE
feat(keymgmt): add new annotation to validate a value with a list of …

### PR DESCRIPTION
**Pull request by steinerda**
…platform keys

### CODE REVIEW

File: sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/validator/PlatformKey.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 @Retention(RUNTIME)
 @Constraint(validatedBy = KeyValidator.class)
 @Documented
+@Deprecated()
```
Comment (christopher-cudennec) [inline sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/validator/PlatformKey.java:29]: Do we really want to deprecate the single key? I can imagine that you want to use exactly one key in many cases. Is it much work maintaining both annotations?

Comment (steinerda) [reply sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/validator/PlatformKey.java:29]: no, it is ok for me. it's not extra work

**Comment by christopher-cudennec** [comment]
With regards to the code duplication issue:
What about extending `KeysValidator` by `KeyValidator`?
If `keyNames` was accessible, `KeyValidator` could set `super.keyNames = new String[] { keyName }`. It would not need to implement `isValid` again.

**Review by christopher-cudennec — COMMENTED** [review]
Looks good to me! ⭐ 
Anyone else wants to review? @SDA-SE/sda-commons-maintainer

**Review by christopher-cudennec — APPROVED** [review]

**Pull request merged**